docs/mp/tracing/01_tracing.adoc

```diff
@@ -94,3 +107,61 @@ Response response = client.target(serviceEndpoint)
     .get();
 ----
 
+include::../../shared/tracing/common-config.adoc[]
+
+==== Configuration using MP Config
+
+Tracing configuration can be defined in `microprofile-config.properties` file.
+
+[source, properties]
+.Tracing configuration
+----
+tracing.components.web-server.spans.0.name="HTTP Request"
+tracing.components.web-server.spans.0.logs.0.name="content-write"
+tracing.components.web-server.spans.0.logs.0.enabled=false
+----
+
+==== Path based configuration in Helidon Web Server
+
+For Web Server we have a path based support for configuring tracing, in addition
+to the configuration described above.
+
+Configuration of path can use any path string supported by the
+Web Server. The configuration itself has the same possibilities
+as traced configuration described above. The path specific configuration
+will be merged with global configuration (path is the "newer" configuration, global is the "older")
+
+[source,properties]
+.Configuration properties
+----
+tracing.paths.0.path="/favicon.ico"
+tracing.paths.0.enabled=false
+tracing.paths.1.path="/metrics"
+tracing.paths.1.enabled=false
+tracing.paths.2.path="/health"
+tracing.paths.2.enabled=false
+tracing.paths.3.path="/greet"
```

Comment:
Examples above are OK, as they all use webserver even in MP.
I would remove the `/greet` option (index 3 and 4).
Have you validated this actually works when used in MP?

Reply:
Yep, they worked on my setup.
Index 3 and 4 are removed.